### PR TITLE
Fix session lens

### PR DESCRIPTION
**Pull request by jackos**
First off just want to thank you for this brilliant vim config, huge productivity boost which would have taken months to figure out.

Just fixing a couple of warnings and errors, one from a deprecation message that was added to cmp.lua a few days ago here: [Pull Request](https://github.com/hrsh7th/nvim-cmp/pull/561/files), and another that occurs sometimes when auto-session wasn't loading before session-lens.

Thanks again this is a legendary neovim config.

### CODE REVIEW

File: config/init.vim [hunk 1]
```diff
@@ -41,6 +41,9 @@ endif
 " let g:loaded_zip = 1
 " let g:loaded_zipPlugin = 1
 
+autocmd BufWritePre *.go lua goimports(1000)
```
Comment (stonemaster) [inline config/init.vim:44]: Those changes seem unrelated

File: config/mappings.vim [hunk 2]
```diff
@@ -24,6 +24,11 @@ endif
 " Navigation {{{
 " ----------
 
+""""""""""""""""""""""""""""""""""""""""""""
```
Comment (stonemaster) [inline config/mappings.vim:27]: Those changes seem unrelated

File: config/mappings.vim [hunk 3]
```diff
@@ -129,10 +134,10 @@ xnoremap <S-Tab> <gv
 " nmap <<  <<_
 
 " Drag current line/s vertically and auto-indent
```
Comment (stonemaster) [inline config/mappings.vim:136]: Those changes seem unrelated

File: config/mappings.vim [hunk 4]
```diff
@@ -242,8 +247,8 @@ nnoremap g1 <cmd>tabfirst<CR>
 nnoremap g5 <cmd>tabprevious<CR>
 nnoremap g9 <cmd>tablast<CR>
 
-nnoremap <A-j>     <cmd>tabnext<CR>
-nnoremap <A-k>     <cmd>tabprevious<CR>
+nnoremap <A-l>     <cmd>tabnext<CR>
```
Comment (stonemaster) [inline config/mappings.vim:250]: Those changes seem unrelated

File: config/plugins.local.yaml [hunk 5]
```diff
@@ -0,0 +1 @@
+- repo: drewtempelmeyer/palenight.vim
```
Comment (stonemaster) [inline config/plugins.local.yaml:1]: This file shouldn't be comitted

File: config/vimrc [hunk 6]
```diff
@@ -4,6 +4,11 @@
 " Runtime and Plugins
 " ===
 
+augroup MyColorSchemeOverrides
```
Comment (stonemaster) [inline config/vimrc:7]: Those changes seem unrelated

File: lua/lsp/gopls.lua [hunk 7]
```diff
@@ -23,6 +23,36 @@ local config = {
 	},
 }
 
+function goimports(timeout_ms)
```
Comment (stonemaster) [inline lua/lsp/gopls.lua:26]: Those changes seem unrelated

File: lua/plugins/cmp.lua [hunk 8]
```diff
@@ -8,7 +8,7 @@ _G.cmp_source_list = function(arr)
 	local config = {
 		buffer = {
 			name = 'buffer',
-			-- opts = {
+			-- options = {
```
Comment (stonemaster) [inline lua/plugins/cmp.lua:11]: Those changes seem unrelated

File: lua/plugins/cmp.lua [hunk 9]
```diff
@@ -24,7 +24,7 @@ _G.cmp_source_list = function(arr)
 		path  = { name = 'path' },
 		emoji = { name = 'emoji' },
 		vsnip = { name = 'vsnip' },
-		tmux  = { name = 'tmux', opts = { all_panes = true }},
+		tmux  = { name = 'tmux', options = { all_panes = true }},
```
Comment (stonemaster) [inline lua/plugins/cmp.lua:27]: Those changes seem unrelated

**Review by stonemaster — CHANGES_REQUESTED** [review]

**Comment by rafi** [comment]
Session lens fixed on 07aa8dea. Thanks though

**Comment by jackos** [comment]
Whoops committed my personal changes to the wrong branch, cheers I'll merge your changes, thanks again for the great project.
